### ISSUE TITLE
Dashboard offline due to scheduled server restart

### ISSUE THREAD

**Issue by thomashlvt** [interval/interval-py#1]
We are running a Python Interval worker through the `interval.listen()` command.

After a scheduled Interval server restart, the dashboard remains offline. The interval worker provides the following log message:
```
[Interval] Failed to connect to Interval (code 1012). Reason: Closing briefly for scheduled server restart.
```
After this it seems 'stuck', it can no longer process new tasks, nor exits.

I would expect one of the following:
- The worker automatically reconnects
- The worker exits with an error status (and our container orchestrator will restart the worker)

**Comment by jacobmischka** [comment]
Thank you for letting us know. Option 1 that you listed is the intended behavior, we'll look into why that's not working correctly for you.

Does this happen frequently? Does it ever properly reconnect afterward for you?

**Comment by jacobmischka** [comment]
We pushed an update to the SDK, we believe this should be fixed in `1.5.1`. Please let us know if you still experience the issue on the updated release.

Thank you for letting us know!

**Comment by thomashlvt** [comment]
The problem has been resolved, thx!
